ui/StatusQ/src/StatusQ/Components/StatusDraggableListItem.qml

```diff
@@ -272,10 +287,10 @@ AbstractButton {
 
     background: Rectangle {
         implicitHeight: 76 // ProfileUtils.defaultDelegateHeight
-        color: root.changeColorOnDragActive && !root.customizable? Theme.palette.alphaColor(Theme.palette.baseColor2, 0.7) : root.bgColor
-        border.width: root.customizable ? 0 : 1
+        color: root.changeColorOnDragActive && root.drawBackground ? Theme.palette.alphaColor(Theme.palette.baseColor2, 0.7) : root.bgColor
```

Comment:
This still renders the background with `root.bgColor`, even if `drawBackground` is `false`; adjust the docu?

Reply:
Yeah, that's true. I've changed the name to better reflect what that property actually alters.